### PR TITLE
Add compat data for SVG <textPath>

### PR DESCRIPTION
**Pull request by JeremiePat**
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/textPath

### CODE REVIEW

File: svg/elements/textpath.json [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "3",
+              "notes": [
+                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: Please replace "re-render" by "re-rendered" and the apostroph at the end by a closing bracket in both Safari notes.

File: svg/elements/textpath.json [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: `startOffset` is supported in IE and Edge, so `true` for IE and `12` for Edge.

File: svg/elements/textpath.json [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "side": {
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: The current versions of Firefox, Chrome, Opera, IE 11, and Edge [don't seem to support this attribute](https://codepen.io/anon/pen/KogLXp).

File: svg/elements/textpath.json [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "method": {
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: The current versions of Firefox, Chrome, Opera, IE 11, and Edge [don't seem to support this attribute](https://codepen.io/anon/pen/xWENLL).

File: svg/elements/textpath.json [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+              ]
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "href": {
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: Not sure all browsers actually support `href` in addition to `xlink:href` yet, though at least the current versions of Firefox, Chrome, Opera, IE, and Edge do so.

File: svg/elements/textpath.json (hunk 3 above)
Comment (JeremiePat) [reply svg/elements/textpath.json]: Yes, I look forward for the Firefox patch to be landed: https://bugzil.la/1446650 :)

File: svg/elements/textpath.json (hunk 5 above)
Comment (JeremiePat) [reply svg/elements/textpath.json]: Safari does too so I think it safe to assume support for all browsers.

**Comment by JeremiePat** [comment]
All fixed :)

File: svg/elements/textpath.json [hunk 6]
```diff
@@ -38,13 +38,13 @@
             "safari": {
               "version_added": "3",
               "notes": [
-                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
+                "Until Safari 6, textPath was not re-rendered when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>"
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: The closing bracket at the end of the string is still missing.

File: svg/elements/textpath.json [hunk 7]
```diff
@@ -38,13 +38,13 @@
...
               ]
             },
             "safari_ios": {
               "version_added": "3",
               "notes": [
-                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
+                "Until Safari 6, textPath was not re-rendered when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>"
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: Same here, closing bracket missing.

File: svg/elements/textpath.json [hunk 8]
```diff
@@ -302,10 +302,10 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": null
+                "version_added": true
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: This should actually be `12`, as well as everywhere else where it currently says `true` for Edge (and probably Edge mobile, too).

File: svg/elements/textpath.json [hunk 9]
```diff
@@ -0,0 +1,390 @@
...
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/textpath.json]: Again, `true` for IE, too.

**Comment by JeremiePat** [comment]
It should be okay now.

**Pull request merged**